deployment/ansible/roles/configure/vars/validator.yml

```diff
@@ -29,6 +29,8 @@ config:
     skip_timeout_commit: true
```

Comment:
Should we have `skip_timeout_commit: false` (default value)?

Reply:
Also please make sure that https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/docs/deployment-design-aws.md contains up-to-date configs